src/debug/Debug.js

```diff
@@ -0,0 +1,112 @@
+import dat from 'dat.gui/build/dat.gui';
+import LayeredMaterial from 'Renderer/LayeredMaterial';
+import Chart from 'chart.js/src/chart';
```

Comment:
Chart.js' `package.json` has `"main": "src/chart.js",` so this should just be `from 'chart.js'`, with the exact same behavior in practice.